src/menu/types.js

```diff
@@ -5,6 +5,8 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
+/* eslint-disable flowtype/no-weak-types */
```

Comment:
How about disabling it only for a specific line? `/* eslint-disable-next-line flowtype/no-weak-types */`